iOSClient/Main/Collection Common/NCSelectableNavigationView.swift

```diff
@@ -80,15 +80,15 @@ extension NCSelectableNavigationView {
             self.selectOcId.removeAll()
             self.selectIndexPath.removeAll()
             self.setNavigationLeftItems()
-            self.setNavigationRightItems(enableMoreMenu: true)
+            self.setNavigationRightItems(toggleMenu: true)
             self.collectionView.reloadData()
         }
     }
 
     func collectionViewSelectAll() {
         selectOcId = selectableDataSource.compactMap({ $0.primaryKeyValue })
         collectionView.reloadData()
-        setNavigationRightItems(enableMoreMenu: true)
+        setNavigationRightItems(toggleMenu: false)
```

Comment:
Keep this as `enableMoreMenu`... `toggleMenu` indicates the menu will hide or show depending on state. We already have a `toggleSelect` function.